atomic_reactor/util.py

```diff
@@ -752,10 +752,16 @@ def get_config_from_registry(image, registry, digest, insecure=False,
         image, registry, digest=digest, insecure=insecure,
         dockercfg_path=dockercfg_path, version=version)
     response.raise_for_status()
+    config_digest = response.json()['config']['digest']
+
+    config_response = query_registry(
+        image, registry, digest=config_digest, insecure=insecure,
+        dockercfg_path=dockercfg_path, version=version)
```

Comment:
I just realised -- this will try to fetch /manifests/$config_digest but that's not what we want.

The config.digest in the manifest is a *blob* digest.

i.e. we want to do the equivalent of this:
```
curl -H "Accept: application/vnd.docker.distribution.manifest.v2+json" \
  $REGISTRY/v2/$REPO/manifests/$MANIFEST_DIGEST | \
  jq -r .config.digest | \
  xargs -i curl $REGISTRY/v2/$REPO/blobs/\{} | \
  jq .
```